examples/acm-terraform-blog-part1/terraform/gke.tf

```diff
@@ -30,8 +30,8 @@ module "enabled_google_apis" {
 }
 
 module "gke" {
-  source             = "terraform-google-modules/kubernetes-engine/google"
-  version            = "~> 31.0"
+  source = "../../.."
+  # [restore-marker]   version            = "~> 31.0"
```

Comment:
Looks like these "leaked" from a test and will need to be reverted.

Reply:
@apeabody I think I got them reverted to the state you want